test.js

```diff
@@ -0,0 +1,70 @@
+var tests = {
+
+    	numbers: [1, 5, 66, 224, 6, 22, 4, 1, 6, 8, 9, 86, 3, 4, 32, 64, 10],
+
+    	/*
+	Given an integer array of size N, compute the sum of all even numbers in this array
+	*/
+	firstTest: function () {
+    		var array = this.numbers.slice();
+    		var result = 0;
+    	
+		for (var i = 0; i < array.length; i++) {
+			if (array[i]%2 == 0) {
+		  		result += array[i];
+		  	}
+		}
+    		return result;
+    	},
+
+    	/*
+	Given an integer array of size N, identify the largest 2 numbers in this array. Do not sort the array.
+	*/
+	secondTest: function () { 
+	 	var array = this.numbers.slice();
+	    	//get the biggest number
+		var biggestNr = Math.max.apply(Math, array);
+			
+		//remove the biggest number from original array
+		var index = array.indexOf(biggestNr);
+		array.splice(index, 1);
+	
+		//get the second biggest number
+		var secondNr = Math.max.apply(Math, array);
+		
+		return biggestNr + ' & ' + secondNr;
```

Comment:
You are doing 2 Math.max witch probably result in looping the array 2 times (not to mention the `indexOf`) can you solve the problem in 1 loop only?